app/scripts/controllers/permissions/index.js

```diff
@@ -206,7 +208,7 @@ export class PermissionsController {
     const approval = this.pendingApprovals.get(id)
 
     if (!approval) {
-      log.error(`Permissions request with id '${id}' not found`)
+      log.debug(`Permissions request with id '${id}' not found`)
```

Comment:
It's not actually a problem if this gets called multiple times, and we _do_ call `rejectPermissionsRequest` over-optimistically. No need for background errors.